### ISSUE TITLE
src package missing for 1.5.0

### ISSUE THREAD

**Issue by ilovezfs** [containous/traefik#2749]
https://github.com/containous/traefik/releases/download/v1.5.0-rc5/traefik-v1.5.0-rc5.src.tar.gz is there

https://github.com/containous/traefik/releases/download/v1.5.0/traefik-v1.5.0.src.tar.gz is not

**Comment by mmatur** [comment]
Every time someone does not respect the issue's template, a little bunny dies.

<details>

![southpark - estrella kill rabbit](https://user-images.githubusercontent.com/5674651/27143914-3cea7896-5130-11e7-81f8-02d8ac2da7a7.gif)
![southpark - estrella kill rabbit2](https://user-images.githubusercontent.com/5674651/27143924-49104c72-5130-11e7-82ae-ccedbd5dfa1c.gif)

</details>
<br>

Please edit your issue description to respect the [issue template](https://raw.githubusercontent.com/containous/traefik/master/.github/ISSUE_TEMPLATE.md) as much as possible.

**Comment by mmatur** [comment]
@ilovezfs Træfik source code is available here https://github.com/containous/traefik/releases/tag/v1.5.0.

The [last one](https://github.com/containous/traefik/archive/v1.5.0.tar.gz) in **Assets** section

**Comment by mmatur** [comment]
I'll close this issue, because I think the question is answered.

**Comment by ilovezfs** [comment]
@mmatur that does not answer why there's sometimes a -src release tarball and sometimes not. In Homebrew, we prefer to use the release tarball when it's available. It's still unclear to me why it's missing right now.

**Comment by ilovezfs** [comment]
Moreover,
```
Josephs-MacBook-Pro:Formula joe$ git log -n14 --format=%h traefik.rb | while read h; do git show $h:Formula/traefik.rb|grep url; done|grep traefik
  url "https://github.com/containous/traefik/releases/download/v1.4.6/traefik-v1.4.6.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.6/traefik-v1.4.6.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.5/traefik-v1.4.5.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.5/traefik-v1.4.5.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.4/traefik-v1.4.4.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.4/traefik-v1.4.4.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.3/traefik-v1.4.3.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.3/traefik-v1.4.3.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.2/traefik-v1.4.2.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.2/traefik-v1.4.2.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.1/traefik-v1.4.1.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.1/traefik-v1.4.1.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.0/traefik-v1.4.0.src.tar.gz"
  url "https://github.com/containous/traefik/releases/download/v1.4.0/traefik-v1.4.0.src.tar.gz"
Josephs-MacBook-Pro:Formula joe$ 
```
So I think something went wrong in your release process this time since the corresponding tarball is suddenly unavailable for 1.5.0

**Comment by ilovezfs** [comment]
@ldez I'm not sure who is responsible for your releases here, but I cannot update `traefik` in Homebrew to 1.5.0 until this is actually sorted out.

**Comment by mmatur** [comment]
@ilovezfs This issue was introduced by #2745. 

And it will be fixed with #2751

**Comment by ilovezfs** [comment]
@mmatur perfect. Thanks :heart:

**Comment by mmatur** [comment]
@ilovezfs Should be good now https://github.com/containous/traefik/releases/download/v1.5.0/traefik-v1.5.0.src.tar.gz

**Comment by ilovezfs** [comment]
@mmatur Voilà https://github.com/Homebrew/homebrew-core/pull/23233

By the way @mmatur @ldez you may want to automate the opening of PRs (using `brew bump-formula-pr`), like the above one I just opened, as part of your release process. See, for example, what Facebook does with `flow` and `yarn`:
https://github.com/Homebrew/homebrew-core/pull/23191
https://github.com/Homebrew/homebrew-core/pull/20188

**Comment by traefiker** [comment]
Closed by #2751.
